delta/plugins/elasticsearch/src/test/scala/ch/epfl/bluebrain/nexus/delta/plugins/elasticsearch/indexing/ElasticSearchDefaultViewsResetterSuite.scala

```diff
@@ -0,0 +1,189 @@
+package ch.epfl.bluebrain.nexus.delta.plugins.elasticsearch.indexing
+
+import cats.effect.{IO, Ref}
+import ch.epfl.bluebrain.nexus.delta.plugins.elasticsearch.ElasticSearchClientSetup
+import ch.epfl.bluebrain.nexus.delta.plugins.elasticsearch.client.IndexLabel
+import ch.epfl.bluebrain.nexus.delta.plugins.elasticsearch.indexing.IndexingViewDef.ActiveViewDef
+import ch.epfl.bluebrain.nexus.delta.plugins.elasticsearch.model.ElasticSearchViewValue.IndexingElasticSearchViewValue
+import ch.epfl.bluebrain.nexus.delta.plugins.elasticsearch.model.permissions
+import ch.epfl.bluebrain.nexus.delta.rdf.syntax.iriStringContextSyntax
+import ch.epfl.bluebrain.nexus.delta.sdk.model.IdSegmentRef
+import ch.epfl.bluebrain.nexus.delta.sdk.views.{IndexingRev, ViewRef}
+import ch.epfl.bluebrain.nexus.delta.sourcing.model.ProjectRef
+import ch.epfl.bluebrain.nexus.delta.sourcing.postgres.Doobie
+import ch.epfl.bluebrain.nexus.delta.sourcing.query.SelectFilter
+import ch.epfl.bluebrain.nexus.testkit.mu.NexusSuite
+import doobie.implicits._
+import fs2.Stream
+import munit.AnyFixture
+
+class ElasticSearchDefaultViewsResetterSuite
+    extends NexusSuite
+    with Doobie.Fixture
+    with ElasticSearchClientSetup.Fixture {
+
+  override def munitFixtures: Seq[AnyFixture[_]] = List(doobie)
+  private lazy val xas                           = doobie()
+
+  private val defaultEsViewId   = "https://bluebrain.github.io/nexus/vocabulary/defaultElasticSearchIndex"
+  private val defaultIndexLabel = IndexLabel.unsafe("default")
+  private val project           = ProjectRef.unsafe("org", "proj")
+
+  test("The reseter should delete scoped events for the default view") {
+    initPartitions(xas, project) >>
```

Comment:
The initPartitions should come in a fixture

Reply:
As it is a setup thing, it would be better if it did not happen in the first test but we can address it in a later PR

Reply:
Ideally it should be in beforeAll, but I'm running into `The fixture `doobie` was not instantiated` when doing that. So putting it as the first test for now...